src/main/kotlin/org/idea_sp/SourcePawnTokenTypeSets.kt

```diff
@@ -40,8 +42,62 @@ interface SourcePawnTokenTypeSets {
             SourcePawnTypes.STATIC_KEYWORD,
             SourcePawnTypes.STOCK_KEYWORD,
             SourcePawnTypes.DECL_KEYWORD,
-            SourcePawnTypes.NEW_KEYWORD,
-            SourcePawnTypes.PUBLIC_KEYWORD
+            SourcePawnTypes.ENUM_KEYWORD,
+            SourcePawnTypes.STRUCT_KEYWORD,
+            SourcePawnTypes.METHODMAP_KEYWORD,
+            SourcePawnTypes.TYPEDEF_KEYWORD,
+            SourcePawnTypes.TYPESET_KEYWORD,
+            SourcePawnTypes.PROPERTY_KEYWORD,
+            SourcePawnTypes.VIEW_AS_KEYWORD
+        )
+        val OPERATORS = TokenSet.create(
```

Comment:
These lists are by no means complete, as well. As I've fiddled with the parser, I've neglected updates to this file. As well, no sort of grand planning came into the organization here.